docs/getting-started/bpfctl-guide.md

```diff
@@ -229,19 +237,211 @@ program's return value. For example, the default `proceed-on` configuration for
+```
+
+Next, load additional eBPF programs that will share the existing maps by passing
+the UUID of the eBPF program that owns the maps using the `--map-owner-uuid`
+parameter:
+
+```console
+bpfctl load-from-file --path /run/bpfd/examples/go-xdp-counter/bpf_bpfel.o -s "stats" --map-owner-uuid 87100e16-4481-4f97-be89-f68d269d6062 xdp --iface vethff657c7 --priority 100
+d6939812-5f6a-42ff-9b55-d3668d8527d0
+```
+
+Use the `bpfctl get` command to display the configuration:
+
+```console
+bpfctl list
+ Kernel ID  Bpfd UUID                             Name   Type  Load Time                
+ 6371       87100e16-4481-4f97-be89-f68d269d6062  stats  xdp   2023-07-18T16:50:46-0400 
+ 6373       d6939812-5f6a-42ff-9b55-d3668d8527d0  stats  xdp   2023-07-18T16:51:06-0400 
+
+bpfctl get 6371
+
+#################### Bpfd State ####################
+
+UUID:                               87100e16-4481-4f97-be89-f68d269d6062
+Path:                               /run/bpfd/examples/go-xdp-counter/bpf_bpfel.o
+Global:                             None
+Map Pin Path:                       /run/bpfd/fs/maps/87100e16-4481-4f97-be89-f68d269d6062
+Map Owner UUID:                     None
+Map Used By:                        d6939812-5f6a-42ff-9b55-d3668d8527d0
+Priority:                           50
+Iface:                              vethff657c7
+Position:                           1
+Proceed On:                         pass, dispatcher_return
+:
+
+bpfctl get 6373
+
+#################### Bpfd State ####################
+
+UUID:                               d6939812-5f6a-42ff-9b55-d3668d8527d0
+Path:                               /run/bpfd/examples/go-xdp-counter/bpf_bpfel.o
+Global:                             None
+Map Pin Path:                       /run/bpfd/fs/maps/87100e16-4481-4f97-be89-f68d269d6062
+Map Owner UUID:                     87100e16-4481-4f97-be89-f68d269d6062
+Map Used By:                        d6939812-5f6a-42ff-9b55-d3668d8527d0
+Priority:                           50
+Iface:                              vethff657c7
+Position:                           0
+Proceed On:                         pass, dispatcher_return
+:
+```
+
+As the output shows, the first program (`87100e16-4481-4f97-be89-f68d269d6062`)
+owns the map, with `Map Program UUID` blank and the `Map Pin Path`
+(`/run/bpfd/fs/maps/87100e16-4481-4f97-be89-f68d269d6062`) that includes its own
+UUID.
+
+The second program (`d6939812-5f6a-42ff-9b55-d3668d8527d0`) references the first
+program via the `Map Program UUID` set to `87100e16-4481-4f97-be89-f68d269d6062`
+and the `Map Pin Path` (`/run/bpfd/fs/maps/87100e16-4481-4f97-be89-f68d269d6062`)
+set to same directory as the first program, which includes the first program's UUID.
+The output for both commands show the map is being used by the second program via
+the `Map Used By` with a value of `d6939812-5f6a-42ff-9b55-d3668d8527d0`.
+
+The eBPF program that owns the maps cannot be unloaded until all eBPF programs
```

Comment:
Yeah based on Dave's earlier comment I'm alright with removing this restriction.